feg/gateway/deploy/roles/feg_services/files/magma_eap_sim.service

```diff
@@ -0,0 +1,29 @@
+# Copyright 2020 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+[Unit]
+Description=Magma EAP SIM FeG service
+
+[Service]
+Type=simple
+ExecStart=/usr/bin/envdir /var/opt/magma/envdir /var/opt/magma/bin/eap_sim -logtostderr=true -v=0
+StandardOutput=syslog
+StandardError=syslog
+SyslogIdentifier=eap_sim
+User=root
+Restart=always
+RestartSec=1s
+StartLimitInterval=0
+MemoryLimit=300M
+
+[Install]
+WantedBy=multi-user.target
+
```

Comment:
are you planning to deploy it with dockerless feg?

Reply:
not planning to, but I think, SIM should have a parity with AKA

Reply:
not planning to, but added for parity with AKA